src/rubrix/client/rubrix_client.py

```diff
@@ -200,6 +205,12 @@ def log(
             processed += response.parsed.processed
             failed += response.parsed.failed
 
+            progress_bar.update(len(chunk))
+        progress_bar.close()
+
+        # TODO: improve logging policy in library
+        print(f"Records logged to {self._client.base_url + '/' + name}")
```

Comment:
Can we add the number of records: print(f"{len(processed} records logged to {self._client.base_url + '/' + name}")

Reply:
por supuesto, let's just keep in mind that this info is also present in the `BulkResponse`.